content/en/news/releases/1.6.x/announcing-1.6.14/index.md

```diff
@@ -19,4 +19,5 @@ This release contains bug fixes to improve robustness. This release note describ
   ([Issue #28916](https://github.com/istio/istio/issues/28916))
 - **Fixed** an issue that caused very high memory usage with a large number of `ServiceEntries`.
   ([Issue #25531](https://github.com/istio/istio/issues/25531))
-
+- **Fixed** an issue in the Stackdriver access log of the `useragent` header.
```

Comment:
I'm not sure this is quite correct. Does Fixed an issue with the `user agent` header in the Stackdriver access log sound correct?

Reply:
Updated.